core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/InstrumentationDirectives.scala

```diff
@@ -2,11 +2,27 @@ package org.broadinstitute.dsde.rawls.metrics
 
 import java.util.concurrent.TimeUnit
 
-import spray.routing.Directive0
+import shapeless.HNil
+import spray.http.Uri
+import spray.routing._
 import spray.routing.directives.BasicDirectives.mapHttpResponse
 import spray.routing.directives.MiscDirectives.requestInstance
+import spray.routing.directives.PathDirectives._
 
 trait InstrumentationDirectives extends RawlsInstrumented {
+
+  // Like Segment in that it matches and consumes any path segment, but does not extract a value.
+  private val SegmentIgnore: PathMatcher0 = Segment.hmap(_ => HNil)
```

Comment:
Oddly Spray doesn't seem to provide something like this. See http://spray.io/documentation/1.2.4/spray-routing/path-directives/pathmatcher-dsl/.

`Neutral` is not quite right because it doesn't consume the input -- so `Neutral / Segment` does not match `foo/bar`.

```
scala> val neutralPm = Neutral / Segment
scala> neutralPm(Uri.Path("foo/bar"))
res2 = Unmatched
scala> val explicitPm = "foo" / Segment
scala> explicitPm(Uri.Path("foo/bar"))
res3 = Matched(,bar :: HNil)
```

Weird. TIL some useless Spray knowledge.